crates/rapid-web/src/util.rs

```diff
@@ -99,3 +100,48 @@ pub fn get_routes_dir(rapid_server_config: Option<&ServerConfig>) -> String {
 		None => panic!("You must have a valid rapid config file in the base project directory!"),
 	}
 }
+
+pub fn get_server_port(config: RapidConfig, fallback_port: u16) -> u16 {
+	let app_type = config.app_type;
+
+	match app_type.as_str() {
+		"server" => match config.server {
+			Some(val) => match val.port {
+				Some(p) => p,
+				None => fallback_port
+			},
+			_ => fallback_port
+		}
+		"remix" => match config.remix {
+			Some(val) => match val.server_port {
+				Some(s_port) => s_port,
+				None => fallback_port
+			},
+			_ => fallback_port
+		}
+		_ => fallback_port
+	}
+}
```

Comment:
We should consider moving this into a helper function to avoid duplicate code

Reply:
Yea -- at some point we need to make a rapid-utils repo so that certain libs that depend on rapid-cli dont have to compile the entire crate when they just need a few helper functions